src/app/zap-templates/zcl/data-model/chip/joint-fabric-datastore-cluster.xml

```diff
@@ -0,0 +1,370 @@
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x0006" name="NodeList" define="NODE_LIST" type="array" entryType="DatastoreNodeInformationEntryStruct">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x0007" name="AdminList" define="ADMIN_LIST" type="array" entryType="DatastoreAdministratorInformationEntryStruct">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x0008" name="Status" define="STATUS_ENTRY" type="DatastoreStatusStruct">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x0009" name="EndpointGroupIDList" define="ENDPOINT_GROUP_ID_LIST" type="array" entryType="DatastoreEndpointGroupIDEntryStruct">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x000A" name="EndpointBindingList" define="ENDPOINT_BINDING_LIST" type="array" entryType="DatastoreEndpointBindingEntryStruct">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x000B" name="NodeKeySetList" define="NODE_KEY_SET_LIST" type="array" entryType="DatastoreNodeKeySetEntry">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x000C" name="NodeACLList" define="NODE_ACL_LIST" type="array" entryType="DatastoreACLEntryStruct">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x000D" name="NodeEndpointList" define="NODE_ENDPOINT_LIST" type="array" entryType="DatastoreEndpointEntryStruct">
+      <access op="read" privilege="administer"/>
+    </attribute>
+
+    <command source="client" code="0x00" name="AddKeySet" isFabricScoped="true">
+      <description>This command SHALL be used to add a KeySet to the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="GroupKeySet" type="DatastoreGroupKeySetStruct"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x01" name="UpdateKeySet" isFabricScoped="true">
+      <description>This command SHALL be used to update a KeySet in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="GroupKeySet" type="DatastoreGroupKeySetStruct"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x02" name="RemoveKeySet" isFabricScoped="true">
+      <description>This command SHALL be used to remove a KeySet from the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="GroupKeySetID" type="int16u"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x03" name="AddGroup" isFabricScoped="true">
```

Comment:
Spec issue: spec has broken text like so:

```
Attempts to add a group with a CAT value of <ref_JFAdministratorCAT,Administrator CAT> or <ref_JFAnchorCAT,Anchor CAT> SHALL fail.
```

That's the rendered text I am quoting.

Also, what does "SHALL fail" mean?  What status code is returned?

Reply:
Tracked by https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620

Reply:
The spec change will be tracked using CHIP-Specifications/connectedhomeip-spec#11622. Once the spec update is complete, an SDK PR can be submitted to align it with the updated specification.